docs/en/integrations/data-ingestion/google-dataflow/dataflow.md

```diff
@@ -0,0 +1,31 @@
+---
```

Comment:
Could add Dataflow as a separate tile on https://clickhouse.com/docs/en/integrations? You need to extend this file https://github.com/ClickHouse/clickhouse-docs/blob/5956b438cdc1e21b138225682da6207fb65001de/docs/en/integrations/index.mdx